generate.js

```diff
@@ -149,7 +153,25 @@ function cli (args) {
     process.exit(1)
   }
 
-  const template = opts.lang === 'ts' || opts.lang === 'typescript' ? typescriptTemplate : javascriptTemplate
+  let template
+  if (opts.lang === 'ts' || opts.lang === 'typescript') {
+    template = typescriptTemplate
+  } else {
+    template = { ...javascriptTemplate }
+
+    if (opts.standardlint) {
+      template.scripts = {
+        ...template.scripts,
+        pretest: 'standard',
+        lint: 'standard --fix'
```

Comment:
In this org we use:

```
lint -> run the linter
lint:fix -> run the linter's fixes
```

Reply:
@Eomm, I prefer such pattern too, but this is for the template, not the org project itself, and for the typescript template it's already just `lint` instead of `lint:fix`, and just `lint` is already what's recommended by the readme right now.

so here I ask two questions:
- should I proceed changing it to `lint:fix`? 
- if yes, then should I update typescript template too (along with its docs) (and readme suggestion)?

Reply:
I reread the readme and I would not change it as it should be already discussed in past